tsdx.config.js

```diff
@@ -6,9 +6,12 @@ module.exports = {
         // Also inline all assets in CSS files using base64 encoding & data URLs.
         config.plugins = [
             styles({
-                autoModules: /index/,
+                autoModules: true,
                 mode: 'extract',
                 url: { inline: true },
+                less: {
+                    globalVars: { primary_color2: '#3f82ef' },
+                },
```

Comment:
For what do we need this?

Reply:
This is a relic of a previous implementation, will remove!